project/Settings.scala

```diff
@@ -135,6 +135,18 @@ object Settings {
     libraryDependencies ++= databaseMigrationDependencies
   ) ++ commonSettings
 
+  val cromwellApiClientSettings = List(
+    name := "cromwell-api-client",
+    libraryDependencies ++= cromwellApiClientDependencies,
+    organization := "org.broadinstitute",
+    scalaVersion := "2.12.1",
```

Comment:
Oh interesting. So this is 2.12 and main cromwell is 2.11?

If so, make sure to use `crossScalaVersions` to publish 2.11 and 2.12 artifacts. I suspect there are others around who might find this useful and they'll likely be on 2.11 for a while.